BlueEastCode/bluerain-platform-reactxp
# Chore/move dependency

## What I did
Move Dependencies into peer dependencies
## How to test
N/A
Is this testable with jest or storyshots?
N/A
Are any of the existing tests are failing ?
N/A
Does this need an update to the documentation?
N/A
Does it have breaking changes ?
N/A
If your answer is yes to any of these, please make sure to include it in your PR.


@bjavaid @amnajunaid please review the changes and merge them.
